test/cql-pytest/test_describe.py

```diff
@@ -970,6 +972,131 @@ def test_hide_cdc_table(scylla_only, cql, test_keyspace):
         for row in ks_tables:
             assert row.table_name == t_name or row.table_name == cdc_log_name
 
+# Verify that the format of the result of `DESC TABLE` targeting a CDC log table
+# has the expected format.
+@pytest.mark.parametrize("test_keyspace",
+                         [pytest.param("tablets", marks=[pytest.mark.xfail(reason="issue #16317")]), "vnodes"],
+                         indirect=True)
+def test_describe_cdc_log_table_format(scylla_only, cql, test_keyspace):
+    with new_test_table(cql, test_keyspace, "p int PRIMARY KEY", "WITH cdc = {'enabled': true}") as table:
+        log_table = f"{table}_scylla_cdc_log"
+        _, log_table_name = log_table.split(".")
+
+        [row] = cql.execute(f"DESC TABLE {log_table}")
+
+        assert row.keyspace_name == test_keyspace
+        assert row.type == "table"
+        assert row.name == log_table_name
+        
+        # The actual content of `row.create_statement` is tested separately in `test_describe_cdc_log_table_layout`.
+        # We only want to confirm here that the statement is wrapped in CQL comment markers.
+        lines = row.create_statement.split("\n")
+        assert lines[0] == "/*" and lines[-1] == "*/"
+
+# Verify that the create statement returned by `DESC TABLE` targeting a CDC log table
+# is correct and as expected.
+@pytest.mark.parametrize("test_keyspace",
+                         [pytest.param("tablets", marks=[pytest.mark.xfail(reason="issue #16317")]), "vnodes"],
+                         indirect=True)
+def test_describe_cdc_log_table_create_statement(scylla_only, cql, test_keyspace):
+    def format_create_statement(stmt: str) -> str:
+        stmt = " ".join(stmt.split("\n"))
+        stmt = " ".join(stmt.split())
+        stmt = stmt.strip()
+        return stmt
+
+    with new_test_table(cql, test_keyspace, "p int PRIMARY KEY", "WITH cdc = {'enabled': true}") as table:
+        log_table = f"{table}_scylla_cdc_log"
+
+        [row] = cql.execute(f"DESC TABLE {log_table}")
+        create_statement = row.create_statement
+
+        # We want to get rid of the CQL comment markers: "/*" and "*/", which make for the first and the last lines.
+        create_statement = create_statement.replace("/*\n", "")
+        create_statement = create_statement.replace("\n*/", "")
```

Comment:
Nit: use `removeprefix` / `removesuffix`. This will make the intention of the code self-explanatory.

https://docs.python.org/3.9/library/stdtypes.html?highlight=removeprefix#str.removeprefix